app/views/users/show.html.erb

```diff
@@ -1,5 +1,10 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <h2><%= "Name: #{@user.name}" %>
+    <% if current_user.id != @user.id %>
+      <%= friendship_status(@user.id) %>
+    <% end %>
```

Comment:
- [ ] move the if statement to a helper method and render line 4 within the helper method instead of having the if statement directly in the view